source/clear-linux/virtual-environments/kvm.rst

```diff
@@ -1,57 +1,156 @@
 .. _kvm:
 
-Using KVM
-#########
+Running Clear Linux as a KVM guest OS
+#####################################
 
-The easiest way to get started in a virtualized environment is to download
-a recent KVM image from the `images`_ directory. Here you'll find a kvm
-image file, the UEFI firmware helper, and the KVM start helper script.
+To run |CL| in a virtualized environment using abbr:`KVM (Kernel-based
+Virtual Machine)`, follow these steps.
 
-Please ensure you have enabled `Intel® Virtualization Technology
-<http://www.intel.com/content/www/us/en/virtualization/virtualization-technology/intel-virtualization-technology.html>`_
-(Intel® VT) and `Intel® Virtualization Technology for Directed I/O
-<https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
-(Intel® VT-d) in your BIOS/UEFI firmware configuration.
+1. Enable `Intel® Virtualization Technology`_ (Intel® VT) and
+   `Intel®Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
+   host machine’s BIOS.
 
-Starter script
-==============
+2. Log in and get root privilege on the host machine::
```

Comment:
Please do not use the :: for commands. Use the code-block directive with the appropriate language, in this case console. Additionally, do not use explicit numbering for ordered lists. use #.
These markup comments apply to the entire document.